cytotable/utils.py

```diff
@@ -188,10 +189,10 @@ def _sqlite_mixed_type_query_to_parquet(
             A str which is a path to a SQLite database file.
         table_name: str:
             The name of the table being queried.
-        chunk_size: int:
-            Row count to use for chunked output.
-        offset: int:
-            The offset for chunking the data from source.
+        page_key: str:
+            The column name to be used as the key for pagination.
+        pageset: Tuple[int, int]:
+            The pageset for chunking the data from source.
```

Comment:
Similar comment here - this is the chunk identifier for some indicator of where the chunk is in context with the rest of the data?

Reply:
Thanks, yes, this is a "set" of the overall "whole" of all keysets for a given table. I'll add more context here to the documentation.